app/presenters/flow_registration_presenter.rb

```diff
@@ -57,6 +57,28 @@ def method_missing(method, *_args, &_block)
     end
   end
 
+  def flows_content
+    content = @flow.nodes.flat_map do |node|
+      case node
+      when SmartAnswer::Question::Base
+        pres = QuestionPresenter.new(node, nil, helpers: [MethodMissingHelper])
+        [pres.title, pres.body, pres.hint]
+      when SmartAnswer::Outcome
+        pres = OutcomePresenter.new(node, nil, helpers: [MethodMissingHelper])
+        [pres.title, pres.body]
+      end
+    end
+
+    content
+      .compact
+      .map do |html|
+        HTMLEntities.new
```

Comment:
Having to strip html out here doesn't seem great, but I don't have a better suggestion, and this'll work for search.

Reply:
I agree, but think that makes more sense than sending this as html.